geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/RasterStore.scala

```diff
@@ -29,6 +28,8 @@ class RasterStore(val rasterOps: RasterOperations) {
 
   def ensureTableExists() = rasterOps.ensureTableExists()
 
+  def ensureBoundsTableExists() = rasterOps.ensureBoundsTableExists()
```

Comment:
As a mental note, we may wish to wrap up functions like ensureTableExists and ensureBoundsTableExists with a function called createTableStructure or something more general.